client/lib/performance-tracking/api.js

```diff
@@ -0,0 +1,43 @@
+const getLux = () => {
+	const lux = window.LUX || {};
+	// Reassign it so changes in LUX are picked when Lux starts
+	window.LUX = lux;
+
+	// Don't send the first navigation automatically, we'll do it.
+	lux.auto = false;
+	lux.debug = true;
```

Comment:
This shows LUX traces in the console. I think we can have this enabled for the proof of concept and disable it if/when we launch in prod.